demo/api/src/db/fixtures/fixtures.console.ts

```diff
@@ -73,130 +54,30 @@ export class FixturesConsole {
-        ];
-
-        let node = await this.pageTreeService.createNode(
-            {
-                name: "Home",
-                slug: "home",
-                attachedDocument: {
-                    id: attachedDocumentIds[0],
-                    type: "Page",
-                },
-            },
-            PageTreeNodeCategory.MainNavigation,
-            scope,
-        );
-        pageTreeNodes.home = node;
-
-        await this.pageTreeService.updateNodeVisibility(node.id, PageTreeNodeVisibility.Published);
-
-        node = await this.pageTreeService.createNode(
-            { name: "Sub", slug: "sub", parentId: node.id, attachedDocument: { id: attachedDocumentIds[1], type: "Page" } },
-            PageTreeNodeCategory.MainNavigation,
-            scope,
-        );
-        pageTreeNodes.sub = node;
-
-        await this.pageTreeService.updateNodeVisibility(node.id, PageTreeNodeVisibility.Published);
-
-        node = await this.pageTreeService.createNode(
-            {
-                name: "Test 2",
-                slug: "test2",
-                attachedDocument: {
-                    id: attachedDocumentIds[2],
-                    type: "Page",
-                },
-            },
-            PageTreeNodeCategory.MainNavigation,
-            scope,
-        );
-        pageTreeNodes.test2 = node;
-
-        await this.pageTreeService.updateNodeVisibility(node.id, PageTreeNodeVisibility.Published);
-
-        node = await this.pageTreeService.createNode(
-            {
-                name: "Test 3",
-                slug: "test3",
-                attachedDocument: {
-                    type: "Page",
-                },
-            },
-            PageTreeNodeCategory.MainNavigation,
-            scope,
-        );
-        pageTreeNodes.test3 = node;
-
-        await this.pageTreeService.updateNodeVisibility(node.id, PageTreeNodeVisibility.Published);
-
-        node = await this.pageTreeService.createNode(
-            {
-                name: "Link1",
-                slug: "link",
-                attachedDocument: {
-                    id: attachedDocumentIds[3],
-                    type: "Link",
-                },
-            },
-            PageTreeNodeCategory.MainNavigation,
-            scope,
-        );
-        pageTreeNodes.link1 = node;
-
-        await this.pageTreeService.updateNodeVisibility(node.id, PageTreeNodeVisibility.Published);
-
-        node = await this.pageTreeService.createNode(
-            {
-                name: "Test Site visibility",
-                slug: "test-site-visibility",
-                attachedDocument: {
-                    id: attachedDocumentIds[4],
-                    type: "Page",
-                },
-            },
-            PageTreeNodeCategory.MainNavigation,
-            scope,
-        );
-        pageTreeNodes.testSiteVisibility = node;
+        const node = await this.pageGeneratorService.generatePage({ name: "Home", scope });
```

Comment:
Could it be confusing that the page generator service doesn't return the page, but returns the page tree node?

Reply:
ah true I think that's confusing. In my opinion it could return both the page and node though

Reply:
I updated it to return both the node and the document